### PR TITLE
Ensure ntp is installed (when Install Recommended Packages is off)

### PR DESCRIPTION
**Pull request by mvidner**
This is a much shorter version ;-) of https://github.com/yast/yast-country/pull/55 and #33.

https://bugzilla.suse.com/show_bug.cgi?id=938951

### CODE REVIEW

File: src/clients/ntp-client_proposal.rb [hunk 1]
```diff
@@ -369,17 +369,11 @@ def Write(param)
       required_package = "ntp"
```
Comment (ancorgs) [inline src/clients/ntp-client_proposal.rb]: Better as constant?

**Comment by ancorgs** [comment]
Apart from a NP comment, looks good to me.

**Comment by kobliha** [comment]
@mvidner NOTLGTM :(

/usr/share/YaST2/modules/Packages.rb.addAdditionalPackage says:

"OBSOLETE! Please, use PackagesProposal::AddResolvables() instead"

**Comment by kobliha** [comment]
There is a reason for that: when users go back, they can decide not to use ntp and then you have to call: PackagesProposal::RemoveResolvables()

**Comment by mvidner** [comment]
@kobliha I can easily replace `Packages.addAdditionalPackage` with `PackagesProposal.AddResolvables`. But without the accompanying `PackagesProposal.RemoveResolvables` it will not be helpful. And the surrounding code has no easy way to accommodate RemoveResolvables, so we would be only papering over the problem.

**Comment by mvidner** [comment]
Thanks for pointing out the obsolete API. I've tried to replace it but it seems that the current PR is better off as is.

**Pull request merged**